servers_v8.json

```diff
@@ -143,5 +143,9 @@
   {
     "name": "Dim.ST",
     "address":["play.simpfun.cn:21402"]
-  }
+  },
+  {
+    "name": "s c p",
```

Comment:
are you sure you might not want this? name is not entirely original, so as full network name might not be ideal
```diff
-    "name": "s c p",
+    "name": "s c p CN",
```

Reply:
or similar renames